tests/graphql_spec.rs

```diff
@@ -193,10 +232,14 @@ fn test_server_to_client_sdl() -> std::io::Result<()> {
   let specs = GraphQLSpec::cargo_read("tests/graphql");
 
   for spec in specs? {
-    let expected = spec.client_sdl;
-    let content = spec.server_sdl[0].as_str();
+    let expected = spec.find_source(SERVER_SDL);
+    let expected = expected.as_str();
+    let content = spec.find_source(CLIENT_SDL);
+    let content = content.as_str();
     let config = Config::from_sdl(content).to_result().unwrap();
+    // error is on the line below
```

Comment:
I figured out the `.graphql` file throwing this error is `/tests/graphql/test-upstream.graphql`; does this make any sense?